spec/services/quotable_service_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe QuotableService do
+  describe 'service to make api calls to quotable api', vcr: { record: :new_episodes } do
```

Comment:
Nice job implementing `:vcr` even though it was not an expectation. Great practice.